mantidproject/mantid
# Missing fractional propagation code in `FractionalRebinning`

During the merge of master into the branch for #20953 the code which propagates the fractional areas in a re-rebin was lost. This PR adds it again. Without this code, the actual example code in #20599 would still give the incorrect errors.

**To test:**

Compile and run the test code in #20599. Check the signal and errors in the two `*_bcksubt_1D_*` workspaces are the same (within a few percent).

Fixes #20599

**Release Notes** 

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
